cs-config/cs_config/functions.py

```diff
@@ -35,32 +35,32 @@ class MetaParams(paramtools.Parameters):
     }
 
 
-def get_inputs(meta_params_dict):
-    '''
-    Function to get user input parameters from COMP
-    '''
+def get_version():
+    return ogusa.__version__    
+
+
+def get_inputs(meta_param_dict):
     meta_params = MetaParams()
-    meta_params.adjust(meta_params_dict)
+    meta_params.adjust(meta_param_dict)
     params = Specifications()
-    spec = params.specification(
-        meta_data=True,
-        serializable=True,
-        year=meta_params.year
-    )
-    return (meta_params.specification(meta_data=True, serializable=True),
-            {"ogusa": spec})
+    # TODO: does OG-USA use year?
+    # params.set_state(year=meta_params.year)
+    return {
+        "meta_parameters": meta_params.dump(),
+        "model_parameters": {
+            "ogusa": params.dump()
+        }
+    }
```

Comment:
I got this error when I tried to do `params.set_state(year=meta_params.year`:
```
E           paramtools.exceptions.ValidationError: {
E               "year": "year is not a valid label."
E           }

../../anaconda3/envs/ogusa-dev/lib/python3.7/site-packages/paramtools/parameters.py:623: ValidationError
```

I don't think OG-USA uses the "year" label. Is that right?

Reply:
That is right - no year label for parameters, but there is a `start_year` variable, that one can change.